packages/cli/src/commands/server/launchDebugger.js

```diff
@@ -52,4 +76,18 @@ function launchChrome(url: string) {
   });
 }
 
-export default launchChrome;
+function launchDebugger(url: string) {
+  if (!commandExists(getChromeAppName())) {
+    logger.info(
+      `For a better debugging experience please install Google Chrome from: ${chalk.underline.dim(
+        'https://www.google.com/chrome/',
+      )}`,
+    );
+    //fallback to default browser
```

Comment:
nit: space here before the inline command. I also think we can rid of that comment and rename the function to something like: `launchDefaultBrowser`

Reply:
Done.